scripts/dev_tools/extract_pr_comments.py

```
@@ -0,0 +1,286 @@
+            "api",
+            f"{endpoint}?per_page=100&page={page}",
+        ]
+        output, code = run_gh_command(args)
+        if code != 0:
+            print(f"Error: API request to {endpoint} page {page} failed.", file=sys.stderr)
+            return items
+
+        try:
+            data = json.loads(output)
+        except json.JSONDecodeError:
+            print(f"Error: Failed to parse JSON from API response.", file=sys.stderr)
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)